src/PreimageCaches.jl

```diff
@@ -107,6 +107,14 @@ function preimage(dom, m::Mapping{S,T}, ::TrivialCache{S,T}, y) where {S,T}
   [x for x in dom if get(m, x, nothing) == y]
 end
 
+function preimage(dom, m::Mapping{S,T}, ::TrivialCache{S,T}, y::Missing) where {S,T>:Union{Missing}}
+  [x for x in dom if ismissing(get(m, x, nothing))]
+end
+
+function preimage(dom, m::Mapping{S,T}, ::TrivialCache{S,T}, y) where {S,T>:Union{Missing}}
+  [x for x in dom if ismissing(get(m, x, nothing)) ? false : get(m, x, nothing) == y]
```

Comment:
I think the conditional can be simplified to

```julia
coalesce(get(m, x, nothing) == y, false)
```

Reply:
done! https://github.com/AlgebraicJulia/ACSets.jl/pull/58/commits/07fb97d5b3d24f6f1b0da0bdfca7ca11f9381a69

Very slick, I haven't seen `coalesce` before.